Assets/Python/GreatPeople.py

```diff
@@ -903,61 +903,105 @@ def setup():
 },
 iCivJapan : {
 	iGreatProphet : [
-		"Koubou Daishi", # 8th
```

Comment:
Can you explain the reasoning behind removing these names?

Reply:
I didn't remove any names from the list, I've only removed Buddhist religious titles in preference of their monk names, following standards in scholarship / religious neutrality. 

- I renamed "Koubou Daishi" (a post-mortem honorific title) to "Kuukai" (his monk name, which is more relevant in historical context)
- "Eisai Zenji" ("Eisai the Zen Master") to "Myouan Eisai"
- "Nikkou Shounin" ("Nikkou the Great") to "Nikkou"
- "Saigyou Houshi" ("Monk Saigyo") to Saigyo

Reply:
Alright, that makes sense. Thanks for the clarification.